package.json

```diff
@@ -185,7 +185,8 @@
     "luxon@^3.0.1": "patch:luxon@npm%3A3.1.0#./.yarn/patches/luxon-npm-3.1.0-16e2508500.patch",
     "improved-yarn-audit@^3.0.0": "patch:improved-yarn-audit@npm%3A3.0.0#./.yarn/patches/improved-yarn-audit-npm-3.0.0-3e37ee431a.patch",
     "lockfile-lint-api@^5.4.6": "patch:lockfile-lint-api@npm%3A5.4.6#./.yarn/patches/lockfile-lint-api-npm-5.4.6-dc86b73900.patch",
-    "symbol-observable": "^2.0.3"
+    "symbol-observable": "^2.0.3",
+    "@metamask/post-message-stream": "^6.1.0"
```

Comment:
This needs to be removed once MetaMask/snaps-monorepo#1082 is merged and released.